### DISCUSSION TITLE
How do I enable a VsCode extension (from the pre-approved list)?

### DISCUSSION THREAD

**Discussion by geoidesic** [codesandbox/codesandbox-client#6665] [Q&A]
I looked in settings – nada. I looked for documentation – couldn't find any for this project at all. [Update: I found documentation but nothing about vscode extensions]
I want to use:

1. `pug`
2. `vetur`

**Comment by JamesACS** [comment]
Hey @geoidesic - Unfortunately you're not able to add your own extensions at this time. If there is an extension that you're interested in we can look at adding this on our end.

**Comment by geoidesic** [comment]
HI @JamesACS as per my original comment

1. `pug`
2. `vetur`

These are already on the pre-approved list (whatever that means).

**Reply by JamesACS** [reply]
Thanks for confirming @geoidesic - I'll pass these through to the team!
